fsspec/tests/test_async.py

```diff
@@ -72,7 +72,7 @@ def test_sync_wrapper_treat_timeout_0_as_none():
 
 
 @pytest.mark.skipif(sys.version_info < (3, 7), reason="no asyncio.run in <3.7")
-def test_throttled_gather(monkeypatch):
+def test_run_coros_in_chunks(monkeypatch):
```

Comment:
Why the rename?

Reply:
It used use asyncio.gather() internally, but now it switched to asyncio.wait so I thought renaming makes sense, though can revert that